Resources/Prototypes/Floof/Entities/Objects/Specific/Hydroponics/equiptments.yml

```
@@ -0,0 +1,23 @@
+- type: entity
+  parent: PlantBag
+  id: ClothingHarvestBag
+  name: Harvest bag
+  description: A bag design by SESWC using bluespace technology to assisted and help botanists everyday needs.
+  components:
+  - type: Sprite
+    sprite: Floof/Objects/Specific/Hydroponics/a_equipment/Harvestbag.rsi
+    state: icon
+  - type: Clothing
+    sprite: Floof/Objects/Specific/Hydroponics/a_equipment/Harvestbag.rsi
+    quickEquip: false
+    slots:
+     - belt
+  - type: Storage
+    grid:
+    - 0,0,19,9
+    quickInsert: true
+    areaInsert: true
+    whitelist:
+      components:
+        - Produce
+        - Seed
```

Code duplication - the values of those fields are already defined in the parent.
```suggestion
  - type: Clothing
    sprite: Floof/Objects/Specific/Hydroponics/a_equipment/Harvestbag.rsi
  - type: Storage
    grid:
    - 0,0,19,9
```